inference-engine/src/gna_plugin/transformations/convert_matmul_to_pointwise_convolution.cpp

```diff
@@ -19,6 +21,18 @@ NGRAPH_RTTI_DEFINITION(ConvertMatmulToPointWiseConvolution, "ConvertMatmulToPoin
 NGRAPH_RTTI_DEFINITION(ConvertMatmulWithBiasToPointWiseConvolution, "ConvertMatmulWithBiasToPointWiseConvolution", 0);
 NGRAPH_RTTI_DEFINITION(ConvertMatmulWithFqToPointWiseConvolution, "ConvertMatmulWithFqToPointWiseConvolution", 0);
 
+static bool BiasValidation(const ngraph::Output<ngraph::Node>& output) {
+    auto bias_output_shape = output.get_node()->get_output_shape(0);
+    if (bias_output_shape.size() == 1) {
+        return bias_output_shape[0] == 1;
```

Comment:
is it possible that bias is 1d but has more than 1 element?

Reply:
I have not met such a case. Do you think that this is impossible?

Reply:
I'm not sure but I believe it's possible case (it can be checked by the test), so we should handle it too (like a case when bias_output_shape.size() > 1 and one of dimensions > 1).

Reply:
added handling of the case bias_output_shape.size() == 1 and bias_output_shape.at(0) > 1